lib/utils.js

```diff
@@ -73,6 +73,12 @@ function normalizeAuthor (_author) {
   return author
 }
 
+function stripNamespaceIfExists (name) {
```

Comment:
```javascript
function stripNamespaceIfExists (name) {
  return name.split('/').slice(-1).join('')
}
```
could also work

Reply:
tried it there, seems to work nicely, also means a cleaner method.